### PR TITLE
[IMP]web: Create real link for many2one in readonly mode so it can be…

### PR DESCRIPTION
**Pull request by msh-odoo**
… opened in new tab

Task: https://www.odoo.com/web#id=32919&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Pad: https://pad.odoo.com/p/r.eeb9580b11688497e5d7a8d563002000

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by msh-odoo** [comment]
LGTM

File: addons/web/static/src/js/fields/relational_fields.js [hunk 1]
```diff
@@ -300,8 +300,8 @@ var FieldMany2One = AbstractField.extend({
     _renderReadonly: function () {
         var value = _.escape((this.m2o_value || "").trim()).split("\n").join("<br/>");
         this.$el.html(value);
-        if (!this.nodeOptions.no_open) {
-            this.$el.attr('href', '#');
+        if (!this.nodeOptions.no_open && this.value) {
+            this.$el.attr('href', _.str.sprintf('#id=%s&model=%s', this.value.res_id, this.field.relation));
```
Comment (mart-e) [inline addons/web/static/src/js/fields/relational_fields.js]: could we add the current menu_id as well? Even if it is not the real menu to lead to that record, it allows to have the same top menu as on the previous page

Comment (msh-odoo) [reply addons/web/static/src/js/fields/relational_fields.js]: After discussion with @ged-odoo we keep it simple as getting current menu_id not simple for instance

**Comment by ged-odoo** [comment]
commit was cherrypicked on 11.0, not on saas-16.

**Comment by ged-odoo** [comment]
https://github.com/odoo/odoo/commit/2bd92cc581463c73760303bba967a0d0fe553786

**Pull request closed without merging**